lightcycle-integration-test/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,28 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.soundcloud.lightcycle.sample.basic">
+
+    <application android:allowBackup="true" android:icon="@mipmap/ic_launcher"
```

Comment:
can you remove the launcher icon and the associated resources ? I think we could test without it